pyseir/load_data.py

```diff
@@ -508,44 +503,185 @@ def load_hospitalization_data_by_state(
         for i, val in enumerate(cumulative[1:]):
             if cumulative[i] > cumulative[i + 1]:
                 cumulative[i] = cumulative[i + 1]
+        return (
+            times_new,
+            hospitalization_data[f"cumulative_{category}"].values.clip(min=0),
+            HospitalizationDataType.CUMULATIVE_HOSPITALIZATIONS,
+        )
+    else:
+        return None, None, None
+
+
+def get_current_hospitalized(state, t0, category):
+    """
+    Return the current estimate for the number of people in the given category for a given US state.
+
+    Parameters
+    ----------
+    state: str
+        US state to lookup.
+    t0: datetime
+        Datetime to offset by.
+    category: str
+        'icu' for just ICU or 'hospitalized' for all ICU + Acute.
+
+    Returns
+    -------
+    time: float
+        Days since t0 for the hospitalization data.
+    current estimate: float
+        The most recent estimate for the current occupied in the requested category.
+    """
+    MIN_DATAPOINTS_TO_CONVERT = 3  # We wait until the third datapoint to have 2 deltas to forecast
+
+    abbr = us.states.lookup(state).abbr
+    df = (
+        combined_datasets.build_us_timeseries_with_all_fields()
+        .get_subset(AggregationLevel.STATE, country="USA", state=abbr)
+        .get_data(country="USA", state=abbr)
+    )
+
+    categories = ["icu", "hospitalized"]
```

Comment:
pull this into a high level constant maybe, `HOSPITALIZATION_CATEGORIES`

Reply:
Yeah. I think this will be part of a bigger refactor. Especially if we pull out the code path for ICU utilization (that goes into the big visualization) vs. hospitalization which goes into the extrapolated days until hospital is overwhelmed.